src/components/Dashboard/index.js

```diff
@@ -234,11 +202,15 @@ class Dashboard extends Component {
   }
 
   renderWidget(marketType) {
-    const {
-      markets, accountShares, accountTrades,
-    } = this.props
-
-    const whitelistedMarkets = markets.filter(market => process.env.WHITELIST[market.creator] && !isMarketResolved(market) && !isMarketClosed(market))
+    const { markets, accountShares, accountTrades } = this.props
+
+    const whitelistedMarkets = markets.filter(market =>
```

Comment:
This is complex. That logic should be moved into a local selector (in selector.js file) and just receive valid markets on the component.

Reply:
Is this going to be implemented now?

Reply:
Please let's do it later when we touch this file for the refactor

Reply:
Ok, sounds good.